### DISCUSSION TITLE
Newbie Testnet issue

### DISCUSSION THREAD

**Discussion by Bunhill-Research** [uniswap-python/uniswap-python#290] [Q&A] [answered]
I have no issues running on mainnet and have tested by approving tokens, however when I try running on görli when I run:

```
node_address= 'https://goerli.infura.io/v3/8edfb1cf5ea64f3fa184935300dbba53'
web3 = Web3(Web3.HTTPProvider(node_address))
uniswapV3 = Uniswap(address=None, private_key=None, version=3, provider=None, web3=web3)
```

I get the following error:

  File "C:\Users\Nick\anaconda3\lib\site-packages\uniswap\uniswap.py", line 109, in __init__
    raise Exception(f"Unknown netid: {self.netid}")

Exception: Unknown netid: 5

Obviously I am probably doing something stupid here, any suggestions?

**Comment by ErikBjare** [comment, marked as answer]
Should work since this was merged: https://github.com/uniswap-python/uniswap-python/pull/261

Are you using the latest version?

**Reply by Bunhill-Research** [reply]
Thank you so much, updating the version fixed it
